psutil/arch/aix/net_connections.c

```diff
@@ -14,6 +14,8 @@
 */
 
 #include "net_connections.h"
+#include "psutil_aix_common.h"
+#include "../../_psutil_common.h"
```

Comment:
Does it break anything if we move "local" includes like these ones after "standard" includes. I mean `#include <fcntl.h>` first (and others), one empty line, then "our" includes.

Reply:
Unfortunately switching the order does break compilation (redefinitions of macros between `pyconfig.h` which is included from `Python.h` with definitions from `/usr/include`. The latter does `ifndef`s and `undef`s so this order is ok).

Reply:
I can make switch the order if I explicitly import `Python.h` first. I'll do that.